test/JDBC/input/views/sys-types-before.sql

```diff
@@ -0,0 +1,88 @@
+select cast(name as varchar(20)) 
```

Comment:
Why do we need this test file if we already have the tests in `sys_types_before_vu_prepare/verify/cleanup` format?

Reply:
Why did you append `before` to its original name? 
Shouldn't it be `sys-types` only?

Reply:
Few test suite in github actions required original sys-types file and few required updated one , so I did this to ensure both.

Reply:
1. Why do we even need `sys-type-before` file if we have the same `sys_types_before_vu_prepare/verify/cleanup` format? These are just extra tests running. 
2. I don't see any jdbc schedule where you have added these tests `sys_types_before_vu_prepare/verify/cleanup`, for upgrade scenarios. Are these even running in any upgrade workflow?

Reply:
1. Actually we were initially having sys-types and sys-types-vu-prepare both initially so I just need renamed it .
2. Yeah we are running updated files now , but I thought to keep older files there , if needed further.
Should  I remove these files ?